prometheus_client/__init__.py

```diff
@@ -4,40 +4,21 @@
     exposition, gc_collector, metrics, metrics_core, platform_collector,
     process_collector, registry,
```

Comment:
Are all of these imports still necessary?

Reply:
There's a chance it causes breakage for some people - currently you can access those packages without explicitly importing them, eg:

```python
import prometheus_client
prometheus_client.metrics.Gauge
```

but if we remove those imports from `__init__.py`, their code would break and need changes to their imports:

```python
import prometheus_client.metrics
prometheus_client.metrics.Gauge
```

Not that there's much reason for people to use those packages instead of the top level, but I thought it wasn't worth risking breakage

Reply:
That's true. Hopefully it is unlikely too many people do that, but I agree there is no reason to break it.